src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -433,8 +430,9 @@ virtenv_setup()
 
     elif test "$virtenv_mode" = "update"
     then
-        test -f "$virtenv/bin/activate" || virtenv_create=TRUE
+        virtenv_create=FALSE
         virtenv_update=TRUE
+        test -f "$virtenv/bin/activate" || virtenv_create=TRUE
```

Comment:
This will lead to a create and then an update. (Which I think also happened before the change)